.github/workflows/cleanup.yaml

```diff
@@ -52,7 +51,7 @@ jobs:
         run: |
           snow sql \
             -q "
-              SET safe_environment_identifier = (SELECT REPLACE('$GITHUB_REF_NAME', '-', '_'));
+              SET safe_environment_identifier = (SELECT REPLACE('${{ github.event.ref }}', '-', '_'));
```

Comment:
I haven't looked very closely, but we will want to be careful to not bring down main / dev databases and tables. Only for feature branches in the future.

Reply:
That could only happen if we delete the `main` branch, which would be catastrophic in other ways. But no need to compound the damage. I added a conditional on this job just in case.